ingestion_server/env.template

```diff
@@ -2,7 +2,7 @@ PYTHONUNBUFFERED="0"
 
 #AWS_REGION="us-east-1"
 
-#ENVIRONMENT="local"
+ENVIRONMENT="local"
```

Comment:
Agreed, I ran into this while trying to test this locally myself.

Reply:
I was thinking about it, and the thing is, often, I have a conflict with default env vars for the catalog. I'm used to using production values as default, but we prefer dev/testing values there. I guess using `local` is not so bad here, and we want to get rid of the ingestion server so that they won't last long anyway.